ui/src/chronograf/components/QueryEditor.js

```diff
@@ -86,37 +87,39 @@ const QueryEditor = React.createClass({
     this.props.actions.groupByTag(this.props.query.id, tagKey);
   },
 
+  handleEditRawText(_queryID, text) {
+    this.props.actions.editRawText(this.props.query.id, text);
```

Comment:
I would say use props and get rid of the first argument OR use the queryID provided by the callback.  I'm leaning towards just using props and removing queryID from the callback in `onUpdate` in the `RawQueryEditor`.